artifact-definition-metaschema.json

```diff
@@ -63,6 +63,38 @@
             ]
           },
           "properties": {
+            "cloud": {
+              "type": "object",
+              "description": "Properties of the cloud supported by Massdriver. Only valid for 'credential' artifact definitions.",
+              "required": [
+                "regions"
+              ],
+              "properties": {
+                "id": {"type": "string", "title": "ID", "description": "Identifier for cloud"},
```

Comment:
This will let us define 'supported clouds' in MD, and be able to find all of the config from artdefs.

Massdriver.MarketPlaceQueries.filter_by_credentials |> where($md.props.id: "aws") [pseudocode]